src/Datasource/ConnectionInterface.php

```diff
@@ -79,4 +79,13 @@ public function logQueries($enable = null);
      * @return object logger instance
      */
     public function logger($instance = null);
+
+    /**
+     * Check if cross talk is supported between two connections
+     *
+     * @param ConnectionInterface $connection Connection to check cross talk with
+     *
+     * @return bool
+     */
+    public function supportsCrossWith(ConnectionInterface $connection);
```

Comment:
Why does this method need to be on the ConnectionInterface? Non-relational datastores will likely never implement this method in a meaningful way.